src/endpoint/s3/s3_rest.js

```diff
@@ -217,7 +217,10 @@ async function authorize_request_policy(req) {
 
     const account = req.object_sdk.requesting_account;
     const account_identifier = req.object_sdk.nsfs_config_root ? account.name.unwrap() : account.email.unwrap();
-    const is_system_owner = account_identifier === system_owner.unwrap();
+    const account_identifier2 = req.object_sdk.nsfs_config_root ? account._id : undefined;
```

Comment:
better have a meaningful name (than account_identifier2).

Reply:
Changed to account_identifier_id.